jsapp/js/account/plans/addOnList.component.tsx

```diff
@@ -0,0 +1,146 @@
+import React, {useCallback, useEffect, useState} from 'react';
+import useWhen from 'js/hooks/useWhen.hook';
+import subscriptionStore from 'js/account/subscriptionStore';
+import type {
+  BasePrice,
+  Organization,
+  Product,
+  SubscriptionInfo,
+} from 'js/account/stripe.types';
+import {
+  isAddonProduct,
+  isChangeScheduled,
+  isRecurringAddonProduct,
+  processCheckoutResponse,
+} from 'js/account/stripe.utils';
+import {postCustomerPortal} from 'js/account/stripe.api';
+import styles from './addOnList.module.scss';
+import BillingButton from 'js/account/plans/billingButton.component';
+
+const AddOnList = (props: {
+  products: Product[] | null;
+  organization: Organization | null;
+  isBusy: boolean;
+  setIsBusy: (value: boolean) => void;
```

Comment:
This is one that I think would be more consistent not using `on…`, since `setIsBusy` never gets used to handle events - see the first part of [this comment](https://github.com/kobotoolbox/kpi/pull/4692#discussion_r1395934036).